### ISSUE TITLE
Will web versions be supported in the future

### ISSUE THREAD

**Issue by ModuleCode** [EIRTeam/EIRTeam.FFmpeg#20]
![image](https://github.com/EIRTeam/EIRTeam.FFmpeg/assets/46812028/71295d90-128c-47b9-bbcc-7204c06a9d14)
I encountered an issue exporting the project to the web

**Comment by EIREXE** [comment]
Yeah the whole ffmpeg extension has to be compiled for web, might actually be easier to just put it in as an engine module if you need it right now, just clone the repo into modules/ffmpeg and hit compile
I'm extremely busy with work right now, but i'll take a look when i can

**Comment by ModuleCode** [comment]
![image](https://github.com/EIRTeam/EIRTeam.FFmpeg/assets/46812028/f7486c43-4fb0-40f0-b5db-434c1e3285e4)
I don’t know why he reported an error
![image](https://github.com/EIRTeam/EIRTeam.FFmpeg/assets/46812028/ff711029-b1e2-4a8d-8e4e-ebc193772cb4)

**Comment by ModuleCode** [comment]
Why does it have to be modules/ffmpeg (Can you tell me why this name must be ffmpeg and where it is configured?)

**Comment by EIREXE** [comment]
That name is  like that because of register_types.cpp functions use just ffmpeg in the name

of course you'll also need a web version of ffmpeg, not sure how one goes about compiling one though

**Comment by ModuleCode** [comment]
![image](https://github.com/EIRTeam/EIRTeam.FFmpeg/assets/46812028/cb8d6a28-d490-4a06-b2a0-14a5ee28a5d6)
I compiled it successfully, but when I played the mp4, it reported an error

**Comment by ModuleCode** [comment]
![image](https://github.com/EIRTeam/EIRTeam.FFmpeg/assets/46812028/398e7476-1c63-411a-ab71-fd989e432a11)
I thought it was a version problem, I tried using version 4.2 but got another error

**Comment by trekfan42** [comment]
> ![image](https://github.com/EIRTeam/EIRTeam.FFmpeg/assets/46812028/398e7476-1c63-411a-ab71-fd989e432a11) I thought it was a version problem, I tried using version 4.2 but got another error

did you ever figure this out?
